packages/local_auth/lib/local_auth.dart

```diff
@@ -73,28 +73,87 @@ class LocalAuthentication {
     AndroidAuthMessages androidAuthStrings = const AndroidAuthMessages(),
     IOSAuthMessages iOSAuthStrings = const IOSAuthMessages(),
     bool sensitiveTransaction = true,
+  }) =>
```

Comment:
I would strongly recommend deprecating this method and then removing it sometime in the future.

Please put a `@deprecated` qualifier to it, remove its dartdoc replacing it with "This method is deprecated, please use `authenticate` with `biometricOnly: true`."

Reply:
Added `@Deprecated` annotation and comment docs